src/align.cpp

```diff
@@ -851,7 +851,7 @@ static void align_same_func_call_params()
    deque<chunk_t *>  chunks;
    deque<AlignStack> as;
    AlignStack        fcn_as;
-   int               idx;
+   size_t            idx;
```

Comment:
If you want to be 100% correct on this one it should be `deque<AlignStack>::size_type`.